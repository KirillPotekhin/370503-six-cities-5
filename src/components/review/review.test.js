import React from "react";
import renderer from "react-test-renderer";
import Review from "./review";

const reviewItem = {
  comment: `Ad quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century.`,
  date: `2019-05-08T14:13:56.569Z`,
  id: 1,
  rating: 4,
  user: {
    avatar: `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/7.jpg`,
    id: 2,
    isPro: false,
    name: `Max`
  }
};

it(`Should Review render correctly`, () => {
  const tree = renderer
    .create(<Review
      reviewItem={reviewItem}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
