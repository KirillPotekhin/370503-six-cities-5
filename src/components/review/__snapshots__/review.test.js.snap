// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Review render correctly 1`] = `
Array [
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Max avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="https://assets.htmlacademy.ru/intensives/javascript-3/avatar/7.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      
    </span>
  </div>,
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Ad quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century.
    </p>
    <time
      className="reviews__time"
      dateTime="2019-04-24"
    >
      May
       
      2019
    </time>
  </div>,
]
`;
